docs/src/running-tests-java.md

```diff
@@ -0,0 +1,88 @@
+---
+id: running-tests
+title: "Running Tests"
+---
+
+Playwright tests can be run in a variety of ways. We recommend hooking it up to your favorite test runner, e.g. [JUnit](./test-runners.md) since it gives you the ability to run tests in parallel, run single tests, etc.
+
+You can run a single test, a set of tests or all tests. Tests can be run on one browser or multiple browsers. By default tests are run in a headless manner meaning no browser window will be opened while running the tests and results will be seen in the terminal. If you prefer you can run your tests in headed mode by using the `--headed` flag.
```

Comment:
"If you prefer you can run your tests in headed mode by using the `--headed` flag." - there is no such thing in java